pkg/kubelet/cpumanager/cpu_manager.go

```diff
@@ -191,6 +214,14 @@ func (m *manager) reconcileState() (success []reconciledContainer, failure []rec
 	success = []reconciledContainer{}
 	failure = []reconciledContainer{}
 
+	if m.IsUnderCPUPressure() {
+		// Make sure all no-CPU pods are evicted.
+		// This really should only do something on the first reconcilation
+		// pass after transitioning to UnderPressure conditions, if all
+		// no-CPU pods are evicted successfully.
+		m.evictNoCPUPods()
```

Comment:
Suggest either to handle the error here instead of discarding, or remove the error from the signature.